x-pack/filebeat/input/awss3/config.go

```diff
@@ -292,3 +305,11 @@ func (c config) getFileSelectors() []fileSelectorConfig {
 	}
 	return []fileSelectorConfig{{ReaderConfig: c.ReaderConfig}}
 }
+
+// Helper function to detect if an ARN is an Access Point
+func isValidAccessPointARN(arn string) bool {
```

Comment:
Can we add some unit test for isValidAccessPointARN func?

Reply:
They already are added into `config_test.go`